src/main/java/com/cryptomorin/xseries/XSound.java

```diff
@@ -1210,6 +1207,9 @@ public void run() {
     public void stopSound(@Nonnull Player player) {
         Objects.requireNonNull(player, "Cannot stop playing sound from null player");
 
+        int version = Integer.parseInt(Bukkit.getBukkitVersion().split("-", -1)[0].split("\\.")[1]);
```

Comment:
Extremely inefficient. Check if a 1.10 sound is available using Enums.getIfPresent method and save it as a constant. If there are no new sounds added in 1.10, just get a material with Material.getMaterial()

Reply:
What about this? It would parse the version number into a byte once. 

```
private static byte version = Byte.parseByte(Bukkit.getBukkitVersion().split("-", -1)[0].split("\\.")[1]);

    public void stopSound(@Nonnull Player player) {
        Objects.requireNonNull(player, "Cannot stop playing sound from null player");

        if(version < 10) return; //Player#stopSound is not available for v1.9 and lower

        Sound sound = this.parseSound();
        if (sound != null) player.stopSound(sound);
    }

```

Reply:
I have no idea why you're parsing it as a byte and not an integer lol. Why not just use it as a simple boolean constant? We're not using it anywhere else. Also, make it final too.